### PR TITLE
#13 - Add node_modules to .npmignore file

### LINKED ISSUE #13: Add node_modules to .npmignore file
Requirement:
- When create and publishing, the package to NPM it should not include the folder node_modules

### PR DESCRIPTION
**Pull request by thypirate**
Added node_modules to .npmignore

### CODE REVIEW

**Comment by FabyoSK** [comment]
Please provide some description

**Comment by FabyoSK** [comment]
Why do you have `yarn.lock` in your PR?

**Comment by FabyoSK** [comment]
Your commit should be:

ISSUE_NUMBER - ISSUE_TITLE

Example:

https://github.com/chuva-io/sisp_payments_js/issues/17 - Add Portuguese README

**Comment by thypirate** [comment]
Ok i will change the commit message. I didn't provide a description because it would be redundant but i can add one  if you want. As far as the yarn.lock just don't accept the changes.

**Comment by FabyoSK** [comment]
LGTM 🚀

**Comment by thypirate** [comment]
Fixed the commit message and i also provided a description plus reverted the yarn.lock file changes.

**Review by FabyoSK — DISMISSED** [review]

**Review by FredSoares — DISMISSED** [review]

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (lcsfort) [inline .npmignore]: @thypirate Please add a new line to the end of this file

Comment (trismegis7us) [reply .npmignore]: @luis737 can we let this go or can you squash a commit with the change so we can ship this?

Comment (thypirate) [reply .npmignore]: @luis737 its done. Srry for the late response.

**Review by FabyoSK — APPROVED** [review]

**Review by vandro — APPROVED** [review]

**Review by lcsfort — APPROVED** [review]

**Review by FredSoares — APPROVED** [review]

**Comment by lcsfort** [comment]
@thypirate thanks for the contribution 🚀

**Pull request merged**